lib/ecto_job/migrations.ex

```diff
@@ -111,4 +112,31 @@ defmodule EctoJob.Migrations do
       execute("DROP TABLE #{Helpers.qualify(name, prefix)}")
     end
   end
+
+  defmodule AddPriorityToJobTable do
+    @moduledoc """
+    Defines a migration to add priority column to the job queue table.
+    This migration can be run multiple times with different values to update multiple queues.
+    """
+
+    import Ecto.Migration
+
+    @doc """
+    Add the priority column to the job queue table with the given name.
+    """
+    def up(name) do
```

Comment:
I think we should we add a combined index on `:priority` since it will be used for sorting. Or perhaps a combined index on `priority, schedule, id`.

Reply:
@mbuhot Done! Now its faster than before.